ai_bridge/static/src/components/chatter_topbar/chatter_topbar.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+
+    <t
+        t-name="mgmtsystem_nonconformity.ChatterTopbar"
+        t-inherit="mail.ChatterTopbar"
+        t-inherit-mode="extension"
+        owl="1"
+    >
+        <xpath
+            expr="//div[hasclass('o_ChatterTopbar_rightSection') and not(@id='mobileChatterTopbar')]/button[last()]"
```

Comment:
On a clean install of CE 16.0 this file gives an error blocking the system.
Is it possible that the responsive web dependency is missing?
```
odoo.addons.base.models.ir_qweb.QWebException: Error while render the template
odoo-1        | ValueError: Element '<xpath expr="//div[hasclass(&#39;o_ChatterTopbar_rightSection&#39;) and @id=&#39;mobileChatterTopbar&#39;]/button[last()]">' cannot be located in parent view
odoo-1        | Template: web.webclient_bootstrap
odoo-1        | Path: /t/t/t[1]/t[4]
odoo-1        | Node: <t t-call-assets="web.assets_backend" t-css="false"/>
odoo-1        | Compiled code:
odoo-1        | code = None
odoo-1        | template = 'web.webclient_bootstrap'
```

Reply:
there is a bug with web_responsive and I made something to make it work, but it was even worse. I will check it